app/views/layouts/_header.html.haml

```diff
@@ -0,0 +1,42 @@
+%header
+  %div{:class => "logo-navbar"}
+    %div{:class => "logo"}
+      %img{:class => "logo-icon", :src => "#{image_path('logo.png')}"}
+      %figcaption{:id => "logo-text"}EUREKA CAFFE
+
+    %div{:id => "navbar"}
+      %ul{:class => "navbar"}
+        %li{:class => "navbar", :id => "navbar-item"}
+          %a{:href => "#"}HOME
+
+        %li{:class => "navbar", :id => "dot"}•
+
+        %li{:class => "navbar", :id => "navbar-item"}
+          %a{:href => "#"}ABOUT
+
+        %li{:class => "navbar", :id => "dot"}•
```

Comment:
You shoulnd't have multiple elements with the same id. If you need to use it multiple times it should be a class.